tgui/packages/tgui-panel/styles/tgchat/chat-default.scss

```diff
@@ -151,6 +151,10 @@ h2.alert {
   color: #6d2f83;
 }
 
+.chat-emoji img {
+  display: none;
+}
```

Comment:
you only need to declare this in the default file, the others will mirror it

Reply:
fixed